types/link_veth.go

```diff
@@ -0,0 +1,54 @@
+package types
+
+import "fmt"
+
+type RawVEthLink struct {
+	LinkCommonParams `yaml:",inline"`
+	Endpoints        []*EndpointRaw `yaml:"endpoints"`
+}
```

Comment:
maybe use the consistent naming pattern here and do `LinkRawVEth`?
btw, why `Raw`?

Reply:
Because this is just the input format, like everything is raw strings. The one of the next steps should be to implement the non-Raw links. That would be the stuff, where e.g. the simple node name string in the endpoint is resolved and the pointer to the node is store in the actual Link object.
Sticking with the raw format is just fuel for this procedural mess we have infront of us.